pkg/sql/sqlstats/insights/registry.go

```diff
@@ -214,3 +223,110 @@ func newRegistry(
 		testingKnobs: knobs,
 	}
 }
+
```

Comment:
The bodies of `makeTxnInsight` and `makeStmtInsight` are essentially what was previously in the `observeTransaction` and `observeStatement` functions in sslocal.StatsCollector, which converted `sqlstats.Recorded*` objects into the insights type.